src/views/HardwareInstallation.vue

```diff
@@ -380,9 +395,99 @@ export default {
 			showComponentRemoveModal: false,
 			modalComponent: undefined,
 			modalComponentName: "",
+			hasData: false,
+			selectManufacturer: undefined,
 		};
 	},
 	computed: {
+		isLoading() {
+			return this.$store.state.mqtt[
+				"openWB/system/configurable/devices_components"
+			];
+		},
+		getDeviceListGeneric() {
+			var generic_arr = [];
+			for (const element of Object.values(this.isLoading)) {
+				if (element.group.includes("generic")) {
```

Comment:
`String.includes()` macht an dieser Stelle keinen Sinn, da der komplette Inhalt identisch sein muss.
```suggestion
				if (element.group == "generic") {
```
Analog bei den anderen Vergleichen.

Reply:
Ok. Wird angepasst. Danke